src/domain/safe/entities/module-transaction.entity.ts

```diff
@@ -4,15 +4,15 @@ import { Operation } from '@/domain/safe/entities/operation.entity';
 export interface ModuleTransaction {
   blockNumber: number;
   created: Date;
-  data: string | null;
+  data: `0x${string}` | null;
   dataDecoded: DataDecoded | null;
   executionDate: Date;
   isSuccessful: boolean;
-  module: string;
+  module: `0x${string}`;
   moduleTransactionId: string;
   operation: Operation;
-  safe: string;
-  to: string;
-  transactionHash: string;
-  value: string;
+  safe: `0x${string}`;
+  to: `0x${string}`;
+  transactionHash: `0x${string}`;
+  value: string | null;
```

Comment:
As we weren't using the correct types in AJV, this was overlooked. [Module](https://github.com/safe-global/safe-client-gateway/blob/f1fcd62c7aa89bbdf10a2abde9dfe10e7274ecbe/src/domain/safe/entities/schemas/module-transaction.schema.ts#L13) and [multisg](https://github.com/safe-global/safe-client-gateway/blob/f1fcd62c7aa89bbdf10a2abde9dfe10e7274ecbe/src/domain/safe/entities/schemas/multisig-transaction.schema.ts#L29) transactions can have a `null` value, as well as their types ([module](https://github.com/safe-global/safe-client-gateway/blob/f1fcd62c7aa89bbdf10a2abde9dfe10e7274ecbe/src/domain/safe/entities/schemas/module-transaction-type.schema.ts#L13)/[multisig](https://github.com/safe-global/safe-client-gateway/blob/f1fcd62c7aa89bbdf10a2abde9dfe10e7274ecbe/src/domain/safe/entities/schemas/multisig-transaction-type.schema.ts#L13)).

There are a few other areas in this PR where this change was necessary which are marked with another comment.